chapter2/tests/t8/=

```diff
@@ -0,0 +1 @@
+State Recv-Q Send-Q Local Address:Port Peer Address:PortProcess
```

Comment:
What is that and why it is needed in a spearate file? Can we avoid this?

Reply:
> What is that and why it is needed in a spearate file? Can we avoid this?

@kseniyashaydurova 
It was an accident, removed